skypeweb/skypeweb_messages.h

```diff
@@ -21,6 +21,13 @@
 
 #include "libskypeweb.h"
 
+typedef enum
+{
+	IMAGE,
+	VIDEO,
+	UNKNOWN
+} URIType;
```

Comment:
also good to 'namespace' enums in C, in case `IMAGE`, `VIDEO`, or `UNKNOWN` are defined anywhere else

Reply:
I was about to namespace it but forgot. Thanks for reminding! :) Done in latest version.